### PR TITLE
Modifies asking for parameters to show app name

### PR DESCRIPTION
**Pull request by cdrage**
Changes the output of when asking for parameters from

ANSWER >> db_pass (Database Password): test

to

ANSWER => etherpad-app | db_pass (Database Password): test

Closes https://github.com/projectatomic/atomicapp/issues/773

### CODE REVIEW

**Comment by kadel** [comment]
#dotests

**Comment by kadel** [comment]
:tada: that was quick :clap: 

:+1: LGTM

File: atomicapp/utils.py [hunk 1]
```diff
@@ -267,22 +267,23 @@ def run_cmd(cmd, checkexitcode=True, stdin=None):
...
         desc = info["description"]
+        logger.debug(info)
         constraints = None
         if "constraints" in info:
             constraints = info["constraints"]
         while repeat:
             repeat = False
             if "default" in info:
                 value = raw_input(
-                    "ANSWER >> %s (%s, default: %s): " % (what, desc, info["default"]))
```
Comment (dustymabe) [inline atomicapp/utils.py]: change from `>>` to `=>` . Hasn't this changed a couple of times already?

File: atomicapp/utils.py [hunk 2]
```diff
@@ -267,22 +267,23 @@ def run_cmd(cmd, checkexitcode=True, stdin=None):
...
+                    "ANSWER => %s | %s (%s, default: %s): " % (app_name, what, desc, info["default"]))
                 if len(value) == 0:
                     value = info["default"]
             else:
                 try:
-                    value = raw_input("ANSWER >> %s (%s): " % (what, desc))
```
Comment (dustymabe) [inline atomicapp/utils.py]: wonder why this `raw_input()` call needs a try/except and the other one doesn't?

File: atomicapp/utils.py (hunk 1 above)
Comment (cdrage) [inline atomicapp/utils.py]: Yeah... I like the format of using `=>`. What do you think?

Comment (dustymabe) [inline atomicapp/utils.py]: +1

**Comment by dustymabe** [comment]
LGTM.. still would like to know why we need try/except for one raw_input and not the other.

**Comment by cdrage** [comment]
@dustymabe i'll investigate and git blame xD. i'll update this in a different PR.

**Comment by cdrage** [comment]
going to run one more test before merge

**Comment by cdrage** [comment]
#dotests

**Pull request merged**